gmcs / mono dependency on Mac OS X?

Compilation aborts with the error below - is the references 'gmcs' executable the mono compiler? Can this dependency be eliminated?

```
idlpp  -I/Users/user/src/opensplice/etc/idl -l cs -S -o custom-psm /Users/user/src/opensplice/etc/idl/dds_dcps_builtintopics.idl
idlpp  -I/Users/user/src/opensplice/etc/idl -l cs -S -o custom-psm /Users/user/src/opensplice/etc/idl/dds_builtinTopics.idl
gmcs -noconfig -nowarn:1701,1702 -warn:4 -define:DEBUG\;TRACE -debug+ -debug:full -optimize- -out:dcpssacs.netmodule -target:module  ../../code/DDS/Condition.cs ../../code/DDS/DataReaderListener.cs ../../code/DDS/DataWriterListener.cs ../../code/DDS/DdsDcpsEnums.cs ../../code/DDS/DdsDcpsInterfaces.cs ../../code/DDS/DdsDcpsStructs.cs ../../code/DDS/DomainParticipantFactory.cs ../../code/DDS/DomainParticipantListener.cs ../../code/DDS/ErrorInfo.cs ../../code/DDS/GuardCondition.cs ../../code/DDS/PublisherListener.cs ../../code/DDS/StatusCondition.cs ../../code/DDS/SubscriberListener.cs ../../code/DDS/TopicListener.cs ../../code/DDS/WaitSet.cs ../../code/DDS/OpenSplice/ContentFilteredTopic.cs ../../code/DDS/OpenSplice/DataReader.cs ../../code/DDS/OpenSplice/DataReaderListenerHelper.cs ../../code/DDS/OpenSplice/DataWriter.cs ../../code/DDS/OpenSplice/DomainParticipant.cs ../../code/DDS/OpenSplice/DomainParticipantListenerHelper.cs ../../code/DDS/OpenSplice/Entity.cs ../../code/DDS/OpenSplice/FooDataReader.cs ../../code/DDS/OpenSplice/FooDataWriter.cs ../../code/DDS/OpenSplice/MultiTopic.cs ../../code/DDS/OpenSplice/Publisher.cs ../../code/DDS/OpenSplice/PublisherDataWriterListenerHelper.cs ../../code/DDS/OpenSplice/QueryCondition.cs ../../code/DDS/OpenSplice/ReadCondition.cs ../../code/DDS/OpenSplice/SacsSuperClass.cs ../../code/DDS/OpenSplice/Subscriber.cs ../../code/DDS/OpenSplice/SubscriberListenerHelper.cs ../../code/DDS/OpenSplice/Topic.cs ../../code/DDS/OpenSplice/TopicDescription.cs ../../code/DDS/OpenSplice/TopicListenerHelper.cs ../../code/DDS/OpenSplice/TypeSupport.cs ../../code/DDS/OpenSplice/Gapi/BuiltinTopicData.cs ../../code/DDS/OpenSplice/Gapi/Conditions.cs ../../code/DDS/OpenSplice/Gapi/ContentFilteredTopic.cs ../../code/DDS/OpenSplice/Gapi/DataReader.cs ../../code/DDS/OpenSplice/Gapi/DataReaderView.cs ../../code/DDS/OpenSplice/Gapi/DataWriter.cs ../../code/DDS/OpenSplice/Gapi/DcpsGapi.cs ../../code/DDS/OpenSplice/Gapi/DomainParticipant.cs ../../code/DDS/OpenSplice/Gapi/DomainParticipantFactory.cs ../../code/DDS/OpenSplice/Gapi/Entity.cs ../../code/DDS/OpenSplice/Gapi/ErrorInfo.cs ../../code/DDS/OpenSplice/Gapi/FooDataReader.cs ../../code/DDS/OpenSplice/Gapi/FooDataReaderView.cs ../../code/DDS/OpenSplice/Gapi/FooDataWriter.cs ../../code/DDS/OpenSplice/Gapi/FooTypeSupport.cs ../../code/DDS/OpenSplice/Gapi/MetaDataInterface.cs ../../code/DDS/OpenSplice/Gapi/MultiTopic.cs ../../code/DDS/OpenSplice/Gapi/Object.cs ../../code/DDS/OpenSplice/Gapi/Publisher.cs ../../code/DDS/OpenSplice/Gapi/Subscriber.cs ../../code/DDS/OpenSplice/Gapi/Topic.cs ../../code/DDS/OpenSplice/Gapi/TopicDescription.cs ../../code/DDS/OpenSplice/Gapi/TypeSupport.cs ../../code/DDS/OpenSplice/Gapi/WaitSet.cs ../../code/DDS/OpenSplice/Database/DDSDatabase.cs ../../code/DDS/OpenSplice/CustomMarshalers/BaseMarshaler.cs ../../code/DDS/OpenSplice/CustomMarshalers/DataReaderMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/DatabaseMarshaler.cs ../../code/DDS/OpenSplice/CustomMarshalers/DatabaseSequenceMarshaler.cs ../../code/DDS/OpenSplice/CustomMarshalers/GapiMarshaler.cs ../../code/DDS/OpenSplice/CustomMarshalers/ListenerMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/QosContainedMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/QosToplevelMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/SampleMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/SequenceMarshalers.cs ../../code/DDS/OpenSplice/CustomMarshalers/StatusMarshaler.cs ../../code/DDS/OpenSplice/Kernel/DDSKernel.cs dds_dcps_builtintopics.cs dds_builtinTopics.cs Idds_dcps_builtintopicsDcps.cs Idds_builtinTopicsDcps.cs dds_dcps_builtintopicsDcps.cs dds_builtinTopicsDcps.cs dds_dcps_builtintopicsSplDcps.cs dds_builtinTopicsSplDcps.cs
make[5]: gmcs: No such file or directory
make[5]: *** [dcpssacs.netmodule] Error 1
make[4]: *** [link] Error 2
make[3]: *** [sacs.ss_link] Error 2
make[2]: *** [dcps.ss_link] Error 2
make[1]: *** [api.ss_link] Error 2
make: *** [src.ss_link] Error 2
```

--- a/setup/environment/x86.darwin10-dev
+++ b/setup/environment/x86.darwin10-dev
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

--- a/setup/environment/x86.darwin10-release
+++ b/setup/environment/x86.darwin10-release
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

--- a/setup/environment/x86_64.darwin10-dev
+++ b/setup/environment/x86_64.darwin10-dev
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

--- a/setup/environment/x86_64.darwin10-release
+++ b/setup/environment/x86_64.darwin10-release
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

--- a/setup/environment/x86_64.darwin10_clang-dev
+++ b/setup/environment/x86_64.darwin10_clang-dev
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

--- a/setup/environment/x86_64.darwin10_clang-release
+++ b/setup/environment/x86_64.darwin10_clang-release
@@ -12,6 +12,7 @@ gawk_check || ERRS=1
 bison_check || ERRS=1
 flex_check || ERRS=1
 javac_check || ERRS=1
+gmcs_check || ERRS=1
 tao_check || ERRS=1
 jacorb_check || ERRS=1
 openorb_check || ERRS=1

